packages/dropdowMenu/components/DropdownMenuItem.tsx

```diff
@@ -19,6 +21,21 @@ export interface DropdownMenuItemProps extends React.HTMLProps<HTMLDivElement> {
 
 const DropdownMenuItem = (props: DropdownMenuItemProps) => {
   const { isActive, isSelected, index, listLength, disabled, ...other } = props;
+  // Importing these was causing issues with `getCSSVarValue` where it would
```

Comment:
I'm having an issue where the custom properties aren't set on :root yet when I try and use `getCSSVarVal` outside of a component.

It works if I use it inside a component, or if the imported Emotion style is called as a function from the component.

I'd appreciate some help on tracking down the root cause of this :)